config/samples/frr-k8s-basic.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: frrk8s.metallb.io/v1beta1
```

Comment:
New samples file were added under the commit bump, with no explaination.
(I am wondering if they were added being are being used somehow or just added for reference)

Reply:
they are not under the comimt bump:
It's a specific commit for adding samples 
https://github.com/metallb/metallb-operator/pull/463/commits/13c5ff39afa06d47722b83729aaf75980562373a

The commit description explain the reason: those are reflected in the csv samples annotation and operator-sdk doesn't throw a warning